clients/web/src/routes/auth/org.svelte

```diff
@@ -0,0 +1,61 @@
+<script lang="ts" context="module">
+    import type {Load} from "@sveltejs/kit";
+    import cookies from "js-cookie";
+
+    export const load: Load = ({session}) => {
+        if (!session.user) {
+            return {redirect: "/auth/login", status: 302};
+        }
+        return {};
+    };
+</script>
+
+<script lang="ts">
+    import {goto} from "$app/navigation";
+    import {session} from "$app/stores";
+    import {currentUser} from "$lib/api";
+    import {Card, CenteredCardLayout} from "$lib/components";
+
+    import {switchOrganizationMutation} from "../../lib/api/mutations/SwitchOrganization.mutation";
+    import Spinner from "../../lib/components/atoms/Spinner.svelte";
+    import {type SessionUser, constants, extractJwt} from "../../lib/utils";
```

Comment:
These should use `$lib` and ensure that we're loading from barrel files properly